engine/src/hopeit/app/api.py

```diff
@@ -194,6 +195,18 @@ def _event_api(
             }
         }
 
+    def _payload_content_type(arg: PayloadDef) -> Tuple[PayloadDef, str]:
+        dt = arg[0] if isinstance(arg, tuple) else arg
+        if inspect.isclass(dt) and issubclass(dt, BinaryDownload):
+            content_type = getattr(dt, 'content_type', "application/json")
```

Comment:
no need to defailt to application/json if we make BinaryDownload have content-type